Couldn't resolve dependency

Sorry, I'm pretty new to Java tooling so I have no idea if this is an easy fix or not, but builds fail due to dependency resolution

```
> mvn -U package assembly:single
[INFO] Scanning for projects...
[INFO]
[INFO] -----------------< de.elbosso.ui.debug:swinginspector >-----------------
[INFO] Building swinginspector 1.1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
Downloading from EL BOSSO - SNAPSHOTS: https://elbosso.github.io/maven2/log4j/log4j/1.2.17/log4j-1.2.17.pom
Downloading from EL BOSSO - RELEASES: https://elbosso.github.io/maven2-releases/log4j/log4j/1.2.17/log4j-1.2.17.pom
Downloading from central: https://central.maven.org/maven2/log4j/log4j/1.2.17/log4j-1.2.17.pom
Downloading from Boundless Repository: https://repo.boundlessgeo.com/main/log4j/log4j/1.2.17/log4j-1.2.17.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.270 s
[INFO] Finished at: 2021-09-01T18:27:04-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project swinginspector: Could not resolve dependencies for project de.elbosso.ui.debug:swinginspector:jar:1.1.0-SNAPSHOT: Failed to collect dependencies at log4j:log4j:jar:1.2.17: Failed to read artifact descriptor for log4j:log4j:jar:1.2.17: Could not transfer artifact log4j:log4j:pom:1.2.17 from/to central (https://central.maven.org/maven2/): transfer failed for https://central.maven.org/maven2/log4j/log4j/1.2.17/log4j-1.2.17.pom: Unknown host No such host is known (central.maven.org) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

--- a/pom.xml
+++ b/pom.xml
@@ -141,7 +141,7 @@ WENN SIE AUF DIE MOEGLICHKEIT EINES SOLCHEN SCHADENS HINGEWIESEN WORDEN SIND.
         </repository>
         <repository>
             <id>central</id>
-            <url>https://central.maven.org/maven2/</url>
+            <url>https://repo.maven.apache.org/maven2/</url>
             <releases>
                 <enabled>true</enabled>
             </releases>

